homeTask6.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <h1 id="first">Hello World!</h1> 
+        <div class="nav-menu">
+            <h1 >Title of the website</h1>
+            <nav>
+                <h3>Main navigation</h1>
+                <ul>
+                    <li><a href="#">Link 1</a></li>
+                    <li><a href="#">Link 2</a></li>
+                    <li><a href="#">Link 3</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <main class="main-content">
+        <p class="bold-italic">This is a paragraph with <b>bold</b> and <i>italic</i> content</p>
+        <hr>
+        <img src="https://images.pexels.com/photos/9091291/pexels-photo-9091291.jpeg" alt="Bike" width="350px" height="350px" id="bike">
+        <br>
+        <a href="https://www.lipsum.com" target="_blank">LINK</a>
```

Comment:
Hint!

- `target="_blank"` is a special keyword that will open links in a new tab every time.
- `target="blank"` will open the first-clicked link in a new tab, but any future links that share target="blank" will open in that same newly-opened tab.